pkg/publish.proj

```diff
@@ -8,6 +8,7 @@
   <Import Project="$(MSBuildThisFileDirectory)dir.props" />
   <Import Project="$(ToolsDir)PublishContent.targets" />
   <Import Project="$(ToolsDir)versioning.targets" />
+  <Import Project="$(ToolsDir)Build.Common.Props" />
```

Comment:
This will work for our current scenarios but if we ever end up trying to sign somewhere else we might want to move this import into dir.props instead.